std_msgs/String object_relation
std_msgs/String object_entity
std_msgs/String object_color
std_msgs/String object_size
std_msgs/String object_num
std_msgs/String flag
geometry_msgs/Point pointing_gesture
